### DISCUSSION TITLE
Plans for future long-term support releses

### DISCUSSION THREAD

**Discussion by whockens** [openssl/openssl#23674] [General Discussion]
It seems the planned release schedule has changed recently.  Are there any plans yet for what will be done for LTS version?   The 3.0 series support will end 7th September 2026.  From what I see, we expect a new release every 6 months starting this April, which I assume will be 3.4, then 3.5 in 10/24, 3.6 in 4/25, and 3/7 in 10/25.  Will one of these versions become an LTS series?  Is there a page describing these plans that I just haven't found?

**Comment by mattcaswell** [comment]
> From what I see, we expect a new release every 6 months starting this April, which I assume will be 3.4, then 3.5 in 10/24, 3.6 in 4/25, and 3/7 in 10/25.

Yes - except it is possible that one of those could be 4.0 (we haven't decided yet).

> Will one of these versions become an LTS series? Is there a page describing these plans that I just haven't found?

Our LTS policy remains the same for now. This is something that is under review so may change - but for now it remains as is. Under our policy we must designate a new LTS every 4 years (at a maximum). 3.0 was released on 2021-09-07 - so we must designate a new LTS by 2025-09-07 at the latest. Since we now release every April and October one of the releases between now and the one in April 2025 will be the next LTS.

**Comment by t8m** [comment]
> From what I see, we expect a new release every 6 months starting this April, which I assume will be 3.4, then 3.5 in 10/24, 3.6 in 4/25, and 3/7 in 10/25.

The first time-based release to be released in April will be 3.3, so you need to shift the versions by one. Otherwise you have it right.

**Comment by whockens** [comment]
Thank you for clarifying this!
